agent/command/git_test.go

```diff
@@ -541,29 +541,6 @@ func (s *GitGetProjectSuite) TestBuildCommandForPullRequests() {
 	s.Equal("git log --oneline -n 10", cmds[8])
 }
 
-func (s *GitGetProjectSuite) TestBuildCommandForPRMergeTests() {
-	conf := s.taskConfig4
```

Comment:
Adding a real github client makes it difficult to run this test. I removed it since it wasn't testing anything too substantial, but let me know if I should find an alternative